src/vm/hints/secp/bigint_utils.rs

```diff
@@ -37,10 +37,10 @@ pub fn nondet_bigint3(
         }
     };
 
-    let arg = split(value)?;
+    let arg: Vec<BigInt> = split(value)?.to_vec();
 
     vm.segments
-        .write_arg(&mut vm.memory, &res_address, &arg, true, &vm.prime)
+        .write_arg(&mut vm.memory, &res_reloc, &arg, true, Some(&vm.prime))
```

Comment:
Using the `Option` is supposed to make the boolean redundant:
```suggestion
        .write_arg(&mut vm.memory, &res_reloc, &arg, Some(&vm.prime))
```

Reply:
Yes, you are right, didn't catch that
https://github.com/lambdaclass/cleopatra_cairo/pull/322/commits/0b26d6d80b4980192a4c67e88249e0e8c60a3594